src/components/search/data/hooks.js

```diff
@@ -59,6 +60,8 @@ export const useDefaultSearchFilters = ({
 
   const filters = useMemo(
     () => {
+      // return `enterprise_customer_uuids:${prodEnterpriseId}`;
```

Comment:
Comment could be added above here stating purpose of return statement if its for testing purposes (similar to comment about ln 46 in this PR within this file)